docs/mine-hnt/validators/validator-deployment-guide.mdx

```diff
@@ -27,20 +26,21 @@ This guide will cover the end-to-end process of deploying a Helium Validator. It
 To successfully deploy a Helium Validator, you'll need to do the following:
 
 - Install the [Helium CLI Wallet](https://github.com/helium/helium-wallet-rs). You are welcome to build from source, but easiest to download the latest binary release.
-- Deploy a virtual server on a cloud service like AWS, GCP, Azure, or Digital Ocean. Or deploy a physical server on which to run the validator. [The recommended hardware and networking requirements can be found here](/mine-hnt/validators/requirements/).
-
-## Create Testnet Wallet
-
-Once you have your [Helium CLI Wallet](https://github.com/helium/helium-wallet-rs) installed locally, it's time to create your Testnet Wallet. Run the following command to create it. (This command format assumes you're using the executable. If you've built the wallet from source it'll look slightly different.)
 
 :::warning
 
 It is a best practice to create your Helium CLI wallet on a separate machine from the one where you will run your Validator.
 
 :::
 
+- Deploy a virtual server on a cloud service like AWS, GCP, Azure, or Digital Ocean. Or deploy a physical server on which to run the validator. [The recommended hardware and networking requirements can be found here](/mine-hnt/validators/requirements/).
+
+## Create Testnet Wallet
+
+Once you have your [Helium CLI Wallet](https://github.com/helium/helium-wallet-rs) installed locally, it's time to create your Wallet. Run the following command to create it. (This command format assumes you're using the executable. If you've built the wallet from source it'll look slightly different.)
+
 ```
-helium-wallet create basic --network testnet
+helium-wallet create basic
 ```
 
 You'll be prompted to supply a new passphrase to complete it. This is used to encrypt/decrypt the `wallet.key` file, and is needed to sign transactions. **Don't lose it.**
```

Comment:
nit: Not your doing, but I think we should use the word `password` here instead of `passphrase`. People might misinterpret `passphrase` to be a mnemonic phrase and have the wrong expectation of the result.

Reply:
updated.